SuperblocksHQ/superblocks-lab
# Add new template "Voting system"

### Description of the Change

New template "Voting system" where you can vote for specific items which you can add, edit and remove.

### Designs

![img1](https://user-images.githubusercontent.com/17637244/47729373-52dd3500-dc60-11e8-9bc9-82dfaa1a11d0.PNG)
![img2](https://user-images.githubusercontent.com/17637244/47729374-5375cb80-dc60-11e8-9914-84d9e42d5f1c.png)

### Verification Process

1. Create new project
2. Select Voting system template
3. User should be able to:

- Compile and Deploy project
- Add new Item
- Vote 
- Edit
- Delete


### Github Issues

Issue #134
